concordium-node/README.md

```diff
@@ -3,9 +3,7 @@
 * Rust (stable 1.45.2 for using static libraries)
 * binutils >= 2.22
 * cmake >= 3.8.0
-* [flatc](http://google.github.io/flatbuffers/flatbuffers_guide_building.html)
-  commit fec58aa129818ed0c0613a7ec36b55135bf81278, but others around it are
-  likely to work as well (build using CMake and copy to `~/.local/bin`)
+* [flatc](http://google.github.io/flatbuffers/flatbuffers_guide_building.html) >= 1.12.0 (build using CMake and copy to `~/.local/bin`)
```

Comment:
I think we should say that 1.11 and 1.12 are known to work, we don't know if `1.13` will work or not, so `>= 1.12` is probably wrong.